flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/CreditBasedPartitionRequestClientHandlerTest.java

```diff
@@ -161,11 +165,42 @@ public void testReceiveBuffer() throws Exception {
 			assertEquals(1, inputChannel.getNumberOfQueuedBuffers());
 			assertEquals(2, inputChannel.getSenderBacklog());
 		} finally {
-			// Release all the buffer resources
-			inputGate.close();
+			releaseResource(inputGate, networkBufferPool);
```

Comment:
The previous tests refactoring can be done as a separate hotfix commit if you like